src/ui/server/test-mocks/index.ts

```diff
@@ -35,6 +36,7 @@ const mockReq = () => {
       origin: 'https://mock.com',
       host: 'mock-host.com',
     },
+    signedCookies: cookieParser(''),
```

Comment:
Is this required? Doesn't seem so, `signedCookies` isn't used anywhere?

Reply:
Without this I had Typescript error.